AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/smithygenerated/aws_cryptography_primitives/shim.py

```diff
@@ -0,0 +1,210 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+import aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy
+import aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.errors
+import aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.models
+import aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.smithy_to_dafny
+import module_
+import software_amazon_cryptography_primitives_internaldafny_types
+from software_amazon_cryptography_primitives_internaldafny_types import (
+    AESDecryptInput_AESDecryptInput as DafnyAESDecryptInput,
+    AESEncryptInput_AESEncryptInput as DafnyAESEncryptInput,
+    AESEncryptOutput_AESEncryptOutput as DafnyAESEncryptOutput,
+    AesKdfCtrInput_AesKdfCtrInput as DafnyAesKdfCtrInput,
+    DigestInput_DigestInput as DafnyDigestInput,
+    ECDSASignInput_ECDSASignInput as DafnyECDSASignInput,
+    ECDSAVerifyInput_ECDSAVerifyInput as DafnyECDSAVerifyInput,
+    GenerateECDSASignatureKeyInput_GenerateECDSASignatureKeyInput as DafnyGenerateECDSASignatureKeyInput,
+    GenerateECDSASignatureKeyOutput_GenerateECDSASignatureKeyOutput as DafnyGenerateECDSASignatureKeyOutput,
+    GenerateRSAKeyPairInput_GenerateRSAKeyPairInput as DafnyGenerateRSAKeyPairInput,
+    GenerateRSAKeyPairOutput_GenerateRSAKeyPairOutput as DafnyGenerateRSAKeyPairOutput,
+    GenerateRandomBytesInput_GenerateRandomBytesInput as DafnyGenerateRandomBytesInput,
+    GetRSAKeyModulusLengthInput_GetRSAKeyModulusLengthInput as DafnyGetRSAKeyModulusLengthInput,
+    GetRSAKeyModulusLengthOutput_GetRSAKeyModulusLengthOutput as DafnyGetRSAKeyModulusLengthOutput,
+    HMacInput_HMacInput as DafnyHMacInput,
+    HkdfExpandInput_HkdfExpandInput as DafnyHkdfExpandInput,
+    HkdfExtractInput_HkdfExtractInput as DafnyHkdfExtractInput,
+    HkdfInput_HkdfInput as DafnyHkdfInput,
+    KdfCtrInput_KdfCtrInput as DafnyKdfCtrInput,
+    RSADecryptInput_RSADecryptInput as DafnyRSADecryptInput,
+    RSAEncryptInput_RSAEncryptInput as DafnyRSAEncryptInput,
+)
+from typing import Any
+
+from .errors import CollectionOfErrors, OpaqueError, ServiceError
+
+
+import Wrappers
+import software_amazon_cryptography_primitives_internaldafny_types
+import aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.client as client_impl
+
+def _smithy_error_to_dafny_error(e: ServiceError):
+       '''
+       Converts the provided native Smithy-modelled error
```

Comment:
spelling: "modeled"

Reply:
This code was written by British/Indian/Australian person :)

Reply:
lol, i didn't realize that "modelled" was UK English; 
which is ironic because i tend to use "cancelled" instead of "canceled".
that said, "modelled" is marked as a spelling error in Chrome,
whereas "cancelled" is not...